recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.11.13" %}
+{% set version = "2018.11.19" %}
```

Comment:
Let's wait until Monday b/c I would like to hear what @gillins has to say about the `kealib` pin.

Reply:
@gillins confirmed in https://github.com/conda-forge/gdal-feedstock/pull/237 this is ok. So we could get this in.